src/mvDrawings.cpp

```diff
@@ -92,8 +92,8 @@ void mvDrawArrow::draw(ImDrawList* drawlist, float x, float y)
 	if (ImPlot::GetCurrentContext()->CurrentPlot)
 	{
 		drawlist->AddTriangleFilled(ImPlot::PlotToPixels(tpp1), ImPlot::PlotToPixels(tpp2), ImPlot::PlotToPixels(tpp3), _color);
-		drawlist->AddLine(ImPlot::PlotToPixels(tp1), ImPlot::PlotToPixels(tp2), _color, (float)ImPlot::GetCurrentContext()->Mx * _thickness);
-		drawlist->AddTriangle(ImPlot::PlotToPixels(tpp1), ImPlot::PlotToPixels(tpp2), ImPlot::PlotToPixels(tpp3), _color, (float)ImPlot::GetCurrentContext()->Mx * _thickness);
+		drawlist->AddLine(ImPlot::PlotToPixels(tp1), ImPlot::PlotToPixels(tp2), _color, (float)ImPlot::GetCurrentContext()->Style.LineWeight * _thickness);
```

Comment:
`Mx`: Changing `Mx` to `Style.LineWeight` is a breaking change that effectively cancels the feature request #1094. The guy who asked for it will definitely be upset.

I see that `Mx` is no longer there in the ImPlot context. We can still simulate it this way:

```cpp
auto Mx = ImPlot::GetPlotSize().x / ImPlot::GetPlotLimits().Size().x;
```

This gives exactly the same scaling factor as `ImPlot::PlotToPixels` uses; I've tested it, and the drawings look identical to what they were in the old version.

Regarding `Style.LineWeight`, I don't think that adding it in the drawings' outline is a good idea. The old version didn't do that, so if somebody customized `mvPlotStyleVar_LineWeight` on their plots, they will get unexpected results with the new version.

So my proposal is to compute `Mx` as shown above, and multiply `_thickness` by `Mx` only (_not_ by `LineWeight`), as it was in the old version.

BTW, do you have a use for non-scaled line thickness (as in your updated version) in your own app? If so, we probably need to add it as an option; not sure where to put the corresponding flag. Adding it to each draw primitive (like `dpg.draw_arrow(pixel_thickness=True)`) seems to be the most straightforward and the least efficient way. But maybe you're not using the drawing primitives in plots, and we can simply drop the subject?..

Reply:
Nope, I'm not using it, so yes, we can definitely drop this